.github/workflows/release.yaml

```diff
@@ -112,17 +112,29 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_DEPLOY_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      # ensure docs are always deployed after merge of changeset PR
+      # ensure docs API is always deployed after merge of changeset PR
       - name: Get the last commit message and set env vars
         run: echo LAST_COMMIT_MSG=$(git --no-pager log -1 --pretty=%B) >> $GITHUB_ENV
 
-      - name: Decides if Docs should be deployed
+      - name: Decides if Docs API should be deployed
         if: startsWith(env.LAST_COMMIT_MSG, 'ci(release):') && env.RELEASE_VERSION_HIGHER_THAN_LATEST == 'true'
-        run: echo SHOULD_DEPLOY_DOCS=true >> $GITHUB_ENV
+        run: echo DEPLOY_STABLE_DOCS_API=true >> $GITHUB_ENV
+
+      # nightly docs API gets updated on every merge to master
+      # and is configured in the vercel dashboard
+      # stable docs API gets updated only when a new release is published
+      - name: Update Docs API (stable)
+        if: github.ref_name == 'master' && env.DEPLOY_STABLE_DOCS_API == 'true'
+        env:
+          VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+          VERCEL_PROJECT_ID: ${{ secrets.VERCEL_TS_DOCS_API_PROJECT_ID }}
+        run: |
+          pnpm install --global vercel@latest
+          vercel --prod --token=${{ secrets.VERCEL_TOKEN }}
```

Comment:
@arboleya we'll need to add all these secrets to the repo. When we get to supporting multiple LTS versions and all that, this same idea would remain, maybe just modified a bit (or maybe not, depending on how we implement it).

Reply:
For env/secrets, please check it with @mchristopher.

Reply:
@luizstacio can you help here? I don't have Vercel access